src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -1682,5 +1682,208 @@ class C
             End Using
 
         End Function
+
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TargetTypePreselection1() As Task
```

Comment:
I still don't get why we need all these preselection tests here.  This is the test of the command handler.  It's about verifying that certain sequences of user input behave as we like.  THe actual behavior of hte underlying providers is not what we're really trying to test here.  I feel like we should have 2 or 3 tests max here unless MatchPriority is really deeply changing how the command handler works.